src/commands/config/create.ts

```diff
@@ -0,0 +1,180 @@
+import fs from "fs";
+import { error, info, success } from "signale";
+import { Argv } from "yargs";
+import { deployDocumentStore } from "../../implementations/deploy/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as CreatedWalletPath } from "../../implementations/wallet/create";
+import { getLogger } from "../../logger";
+import { highlight } from "../../utils";
+import { handler as CreateTemporaryDns } from "../dns/txt-record/create";
+import ConfigTemplate from "./config-template.json";
+import { CreateConfigCommand } from "./config.type";
+
+interface ConfigFile {
+  wallet: string;
+  forms: Form[];
+}
+
+interface Form {
+  defaults: {
+    issuers: (VerifiableDocumentIssuers | TransferableRecordIssuers)[];
+  };
+}
+interface VerifiableDocumentIssuers {
+  id?: string;
+  revocation?: { type: string };
+  name: string;
+  documentStore?: string;
+  identityProof: {
+    type: string;
+    location?: string;
+    key?: string;
+  };
+}
+interface TransferableRecordIssuers {
+  name: string;
+  tokenRegistry: string;
+  identityProof: {
+    type: string;
+    location?: string;
+  };
+}
+
+const { trace } = getLogger("config:create");
+
+export const command = "create [options]";
+
+export const describe = "Create a config file";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      demandOption: true,
+    })
+    .option("encrypted-wallet-path", {
+      type: "string",
+      description: "Path to file containing wallet.json",
+      normalize: true,
+    })
+    .option("config-template-path", {
+      type: "string",
+      description: "Path to file containing config template",
+      normalize: true,
+      demandOption: true,
+    });
+
+export const handler = async (args: CreateConfigCommand): Promise<void> => {
+  trace(`Args: ${JSON.stringify(args, null, 2)}`);
+  if (!fs.existsSync(args.outputDir)) {
+    fs.mkdirSync(args.outputDir);
+  }
+
+  try {
+    let walletPath = "";
+    if (!args.encryptedWalletPath) {
+      info(`Please enter password to create wallet`);
+      const createWalletParams = {
+        outputFile: `${args.outputDir}/wallet.json`,
```

Comment:
let's keep the bahviour consistent. create wallet expect a full path, not a folder. Remove /wallet.json

Reply:
ok fixed.

Reply:
It's me or I smoked and you were right ?

I said `create wallet expect a full path`, which was the case and ask you to remove `/wallet.json` and now it's just a folder ?

Reply:
i don't understand this part, if we want to create a file in a folder shouldn't we pass in  `${args.outputDir}/wallet.json` in the outputFile so that it will create it in the folder we created as a wallet.json file?